docs/usage.md

```diff
@@ -10,7 +10,13 @@ After the initial backup, use the following commands:
 
 These commands are run on the host you [installed][1] Backup Utilities on.
 
-### Example backup and restore usage
+## Understanding Configuration
+
+Users can supply their own configuration file or use the example configuration file as a template where they can set up their environment for backing up and restoring.
```

Comment:
Since the main heading and the next sentence are both in second person, should we be a little less formal here?